neofs-testlib/neofs_testlib/env/env.py

```diff
@@ -42,6 +43,7 @@ class WalletType(Enum):
 
 
 class NeoFSEnv:
+    _id = f"{datetime.datetime.now()}".replace(" ", "-").replace(":", "-").replace(".", "-")
```

Comment:
Doesn't look reliable, format your date and time manually if you want to be sure in anything.

Reply:
fixed